source/tests/pt_expt/test_training.py

```
@@ -0,0 +1,630 @@
+                    actual_lr,
+                    expected_lr,
+                    places=10,
+                    msg=f"LR after restart should be lr_schedule({trainer2.start_step})"
+                    f"={expected_lr}, got {actual_lr}",
+                )
+
+                trainer2.run()
+
+                # lcurve should have entries appended (restart opens in append mode)
+                with open(os.path.join(tmpdir, "lcurve.out")) as f:
+                    lines = [l for l in f.readlines() if not l.startswith("#")]
```

Ruff (0.15.2)

[error] Ambiguous variable name: `l`

(E741)